code/game/gamemodes/marker/gamemodes.dm

```diff
@@ -143,7 +143,7 @@ Non-critical characters like any ghost-roles you may wish to add, or even antags
 	if(marker_active)	//Marker must be active
 		if (get_historic_crew_total() >= minimum_historic_crew)	//We need to have had a minimum total crewcount
 			var/minimum_living_crew = Ceiling(get_historic_crew_total() * minimum_alive_percentage)	//This many crew players at least, need to be left alive
-			if (get_living_crew_total() < minimum_living_crew)
+			if (get_alive_crewmates_count() < minimum_living_crew)
```

Comment:
Everything is good except the function name, its ambiguous and too similar to the former.

call it get_living_active_crew_aboard_ship
Its a bit of a mouthful but it'd be accurate

Reply:
Welp, I was trying to keep it somewhat compact. That was initial name for it, heh.